libs/vkd3d/command.c

```diff
@@ -17334,11 +17391,15 @@ static void STDMETHODCALLTYPE d3d12_command_queue_ExecuteCommandLists(ID3D12Comm
         {
             if (cmd_list->cmd.iterations[iter].vk_init_commands)
             {
+                cmd_cost[j] = VKD3D_COMMAND_COST_HIGH;
```

Comment:
Do we really do significant work in init command lists?

Reply:
If there's ExecuteIndirect memes involved then apparently yes. Not fusing init commands destroys frame pacing in FF16 at least.

Reply:
Do we know what happens in the init command lists? I can't think of anything expensive unless it's doing DGC.

Reply:
The only things I can find that would use init command buffer is:

- Query pool reset (should be virtually free)
- DGC fixups (should still be cheap)
- Emitting predicated commands

Could there an off-by-one error where the cost metric is wrong and we confuse init buffer for a real command buffer?